docs/site.json

```diff
@@ -20,11 +20,15 @@
     }
   ],
   "plugins" : [
-    "filterTags"
+    "filterTags",
+    "disqus"
```

Comment:
hmm not sure if its necessary for our docs (unless just for review here).

We should direct users to the github page

Reply:
Yup, it was just for review. Will remove it :-)